app.js

```diff
@@ -27,7 +27,7 @@ app.get('/', (req, res) => res.render('index'));
 
 const discover = require('./routes/discover')
 app.get('/discover', discover.view);
-app.post('/discover', discover.getOrganizations);
+app.get('/discover/:filter', discover.getOrganizations);
```

Comment:
Is this an optional parameter? If so you can mark it as optional `:filter?`

Reply:
It's not optional at the moment, as to get all results you pass in "all." The filter string is grabbed directly from a dropdown on the page.